static/sass/custom/_search-results.scss

```diff
@@ -30,6 +32,55 @@
     @media (min-width: $breakpoint-small) {
       display: flex;
     }
+
+    &--link {
+      color: $color-dark;
+    }
+
+    .p-contextual-menu {
+      display: block;
+
+      &__toggle {
+        text-transform: capitalize;
+
+        &--active {
+          border-color: $color-focus;
+        }
+      }
+
+      &__dropdown {
+        $tooltip-size: 8px;
+        min-width: 19rem;
```

Comment:
The small screen support is much improved but... 

<img width="310" alt="Screen Shot 2019-05-06 at 10 40 07 AM" src="https://user-images.githubusercontent.com/532033/57240363-80c67c00-6feb-11e9-8365-93c16563ee2f.png">

To fix this, if this `min-width` is only used on the `mid` and `large` breakpoints then it'll allow the select dropdown to scale correctly on the small screen, the Ubuntu series's will wrap, and be the proper width on wider screens.

<img width="302" alt="Screen Shot 2019-05-06 at 10 47 25 AM" src="https://user-images.githubusercontent.com/532033/57240678-5aeda700-6fec-11e9-8ec3-5f0969915488.png">

<img width="516" alt="Screen Shot 2019-05-06 at 10 47 53 AM" src="https://user-images.githubusercontent.com/532033/57240707-6b9e1d00-6fec-11e9-90de-d1929c49cc59.png">